tools/ci_build/build.py

```diff
@@ -981,8 +981,8 @@ def main():
                 mkldnn_run_onnx_tests(build_dir, configs, onnx_test_data_dir)
 
         # run nuphar python tests last, as it installs ONNX 1.5.0
-        if args.enable_pybind and not args.skip_onnx_tests and args.use_nuphar:
-            nuphar_run_python_tests(build_dir, configs, args.azure_sas_key)
+        #if args.enable_pybind and not args.skip_onnx_tests and args.use_nuphar:
+        #    nuphar_run_python_tests(build_dir, configs, args.azure_sas_key)
```

Comment:
The test on benchmark script covers LSTM-to-Scan/symbolic shape inference/quantization scripts, and should not be disabled. It should take less than 1 min to run. I am OK to disable it for now, but it should eventually be enabled after fixed issues.

Reply:
Feel free to enable it again.